tests/tests.py

```diff
@@ -30,7 +30,7 @@
 _non_parallel = True
 if platform.python_version() < '3':
     _non_parallel = False
-
+    import mock
```

Comment:
You cannot import mock without adding it as a dependency for python versions < 3.3.